modules/document_repository/php/files.class.inc

```diff
@@ -65,87 +48,64 @@ class Files extends \NDB_Page
      *
      * @return ResponseInterface The outgoing PSR7 response
      */
-    public function handle(ServerRequestInterface $request) : ResponseInterface
+    public function handle(ServerRequestInterface $request): ResponseInterface
     {
         $config = \NDB_Factory::singleton()->config();
         $path   = \Utility::appendForwardSlash(
             $config->getSetting("documentRepositoryPath")
         );
         switch ($request->getMethod()) {
         case "POST":
-            if ($this->uploadDocFile($request)) {
-                return (new \LORIS\Http\Response())
-                    ->withHeader("Content-Type", "text/plain")
-                    ->withStatus(200)
-                    ->withBody(
-                        new \LORIS\Http\StringStream(
-                            json_encode("uploaded successfully")
-                        )
-                    );
-            }
-            return new \LORIS\Http\Response\JSON\Conflict("duplicate file name");
+            return $this->uploadDocFile($request);
         case "PUT":
             $tmp = $request->getBody();
             $arr = (array)json_decode($tmp);
-            $this->editDocFile($arr);
-            return (new \LORIS\Http\Response())
-                ->withHeader("Content-Type", "text/plain")
-                ->withStatus(200)
-                ->withBody(
-                    new \LORIS\Http\StringStream(
-                        json_encode("file updated")
-                    )
-                );
+            return $this->editDocFile($arr);
         case "DELETE":
             $id = basename($request->getUri()->getPath());
             $this->deleteFile($id);
-            return (new \LORIS\Http\Response())
-                ->withHeader("Content-Type", "text/plain")
-                ->withStatus(200)
-                ->withBody(
-                    new \LORIS\Http\StringStream(
-                        json_encode("file deleted")
-                    )
-                );
+            return (new \LORIS\Http\Response\JSON\OK(
+                ['message' => 'File deleted.']
```

Comment:
@johnsaigle Should the array syntax be the other way?

Reply:
I think we may be changing it #6104 ? If that PR fails then I'll create a PHPCS rule for the long way. Either way this will get automatically cleaned up very soon